recipe/meta.yaml

```diff
@@ -39,6 +42,8 @@ requirements:
     - make           # [not win]
     - cmake >=3.1    # [win]
     - jom            # [win]
+    - binutils       # [linux]
+    - patch          # [linux]
   host:
```

Comment:
On `win` platforms should be:
`    - m2-patch  # [win]`

Reply:
Thanks, added selector, no patching is done on Windows.